src/services/completions.ts

```diff
@@ -1952,6 +1980,40 @@ namespace ts.Completions {
         }));
     }
 
+    function isTypeScriptOnlyKeyword(kind: SyntaxKind) {
+        switch (kind) {
+            case SyntaxKind.AbstractKeyword:

```

Comment:
Can this list be sourced from a single place, so this and the similar list in `getInJsKeywords` don't drift?  IIRC we have a function for converting from keyword string to kind and another for the reverse.

Reply:
The `getInJsKeywords` is for test and we don't want to mix that but have that list explicit just like any other keyword filter tests that we have.